pkgs/servers/monitoring/mtail/default.nix

```diff
@@ -2,16 +2,16 @@
 
 buildGoModule rec {
   pname = "mtail";
-  version = "3.0.0-rc43";
+  version = "3.0.0-rc46";
 
   src = fetchFromGitHub {
     owner = "google";
     repo = "mtail";
     rev = "v${version}";
-    sha256 = "1lw7gz6fh3z7j4hin4q6y3wxxl4sgd4li1c0z60hmvfs671n6dg9";
+    sha256 = "sha256-/PiwrXr/oG/euWDOqcXvKKvyvQbp11Ks8LjmmJjDtdU=";
```

Comment:
Was this hash generated by unstable nix? Can we make this a stable nix hash instead?

Reply:
Stable nix also understands them so it doesn't matter.

Reply:
@glittershark Any reason you prefer stable nix hashes if the unstable ones work?

Note that I get unstable nix hashes when I use `nix-update` (which make updating a package painless for me) on my computer.